src/Kunstmaan/AdminBundle/Helper/IsAdminRouteHelper.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Kunstmaan\AdminBundle\Helper;
+
+/**
+ * Class IsAdminRouteHelper
+ */
+class IsAdminRouteHelper
```

Comment:
I suggest simply `AdminRouteHelper` as in the future it could have not only `isAdminRoute` method.